docs/blog/newsletter-november-2023.rst

```diff
@@ -0,0 +1,123 @@
+
+Distribution methods for docs changelogs naturally depend on the intended audience. For example, you might share an internal-only changelog via the company Slack workspace or other established employee communication channels. For external audiences, the changelog might be published in the docs or posted in the docs repository. 
+
+---------------------
+Doc Team Organization 
+---------------------
+
+The organization of a documentation team can have a significant effect on team members. As a manager, you may want to review your current setup; as a documentarian, you may want to consider the structure before joining an existing team.
+
+One aspect to consider is the ratio of more experienced to less experienced writers. Some teams expect more experienced writers to train less experienced ones. The larger the team is, the more likely it is that writers can be matched up.
+
+Another consideration is whether specific writers have assigned roles (such as writing, editing, reviewing, or automating procedures). Some teams have generalists, who can also support the team with specific skills. Other teams have specific assignments for each member. A manager may be responsible for filling in gaps. 
+
+Besides roles, you may need to consider if writers have responsibilities for the full product line or for specific products (or features). When all writers are familiar with everything, workload balancing can be easier. One person mentioned assigning specific products or features to newcomers, but expecting more experienced writers to be available for all products. 
+
+To complete their work successfully, documentarians interact with others. When assigned to a product, the interactions may be limited to product managers. With features, writers may need to interact with very specific subject matter experts. Through these interactions, documentarians can develop their skills, get known throughout the company, and demonstrate professional growth. 
+
+But what happens when products or product lines change? See `Keeping Up-To-Date with Necessary Changes <https://www.writethedocs.org/blog/newsletter-september-2023/>`__ for dealing with ongoing changes. A manager needs to be aware of significant upcoming changes. With timely communications, writers can develop the skills needed to document new features or products.
+
+----------------------
+Measuring Docs Quality
+----------------------
+
+Evaluating documentation quality is a common practice, but it often presents a unique set of challenges. A recent conversation shed light on the complexities involved in this process, offering insights into how to approach it effectively.
+
+One person shared the dilemma of having to rate around 200 articles in their knowledge base on a scale of 1–5, all within a tight timeframe. The request came from their CTO, who believed it would only take 10 minutes.
```

Comment:
Do we to get this specific with the numbers? Might be worth being more vague here (something I don't say often)

Reply:
Fair point.
```suggestion
One person shared the dilemma of having to numerically rate hundreds of articles in their knowledge base within a tight timeframe. The request came from their CTO, who believed it would only take a few minutes in total.
```

Reply:
Yeah, sounds good :+1: